nornir/core/inventory.py

```diff
@@ -191,6 +191,15 @@ def ssh_port(self):
         """Either ``nornir_ssh_port`` or 22."""
         return self.get("nornir_ssh_port", 22)
 
+    @property
```

Comment:
Is it worth adding this as a property? I think I'd rather keep things in the ssh config to avoid making this more complex. What do you think? Is there a strong case where you think it'd be worth managing this in an inventory file instead of in a proper ssh config file?

Reply:
I thought pretty much the same thing...i.e. that it probably didn't make sense to have a separate attribute for this and that we should just do this through the SSH config file.